templates/contract-deploy/run-contract-setup.sh

```diff
@@ -0,0 +1,185 @@
+#!/bin/bash
+# This script is responsible for deploying the contracts for zkEVM/CDK.
+
+echo_ts() {
+    timestamp=$(date +"[%Y-%m-%d %H:%M:%S]")
+    echo "$timestamp $1"
+}
+
+wait_for_rpc_to_be_available() {
+    rpc_url="$1"
+    counter=0
+    max_retries=20
+    until cast send --rpc-url "{{.l1_rpc_url}}" --mnemonic "{{.l1_preallocated_mnemonic}}" --value 0 "{{.zkevm_l2_sequencer_address}}"; do
+        ((counter++))
+        echo_ts "L1 RPC might not be ready... Retrying ($counter)..."
+        if [ $counter -ge $max_retries ]; then
+            echo_ts "Exceeded maximum retry attempts. Exiting."
+            exit 1
+        fi
+        sleep 5
+    done
+}
+
+fund_account_on_l1() {
+    name="$1"
+    address="$2"
+    echo_ts "Funding $name account"
+    cast send \
+        --rpc-url "{{.l1_rpc_url}}" \
+        --mnemonic "{{.l1_preallocated_mnemonic}}" \
+        --value "100ether" \
```

Comment:
For some reason I though this was a param, but I guess not.

Reply:
You mean the 100 ether?

Reply:
I mean we could make it a param if you want :)